### PR TITLE
feature/project-scope

### PR DESCRIPTION
**Pull request by Ramchy-J**
Adding the the project scope and development standard

### CODE REVIEW

**Comment by eapg** [comment]
The .idea/ files are own of your IDE, they are not need to be committed as they will be different for each developer.

Take out the `.gitignore` file from .idea folder, also ignore all the files inside .idea folder adding them to `gitignore` file. This way those files weren't be commit.

**Comment by Ramchy-J** [comment]
Removing the unnecesary files

**Review by eapg — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+\.idea
```
Comment (reynoldmorel) [inline .gitignore]: nit: Since this is a folder, could change it to be `.idea/`?

**Review by reynoldmorel — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Ramchy-J) [reply .gitignore]: Yes, it can be chaged. I have already done it.

**Pull request merged**